### ISSUE TITLE
Update docs to use string values instead of ints

### ISSUE THREAD

**Issue by nirvdrum** [ansible/ansible#60335]
##### SUMMARY

The documentation for various modules uses bare int values, but these now print a warning message indicating they should be quoted string values. The warning message was already reported as #58347 and closed as not a bug. Since it appears bare int values are no longer acceptable (at least not without a warning), I think the documentation should be updated to avoid use of such values. Incidentally, the original reporter of #58347 broached that topic, but I think it was lost in the shuffle.

##### ISSUE TYPE

- Documentation Report

##### COMPONENT NAME

There are likely others, but the two modules I know that document behavior that will result in a warning message being presented are:

- sysctl
- ufw

##### ANSIBLE VERSION

```paste below
ansible 2.8.3
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/nirvdrum/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15+ (default, Nov 27 2018, 23:36:35) [GCC 7.3.0]
```

##### CONFIGURATION

Default configuration. Nothing changed.

##### OS / ENVIRONMENT

Ubuntu 18.04.

##### ADDITIONAL INFORMATION

The first example from the Ansible sysctl module documentation looks like:

```yaml
# Set vm.swappiness to 5 in /etc/sysctl.conf
- sysctl:
    name: vm.swappiness
    value: 5
    state: present
```

Since this will result in a warning being printed out, that can be confusing for users. It's particularly confusing for anyone that happened to have used the example code with releases of Ansible that previously did not produce a warning. If this is the path going forward, however, I think we can clear things up quite a bit by quoting values in the documentation. E.g.,

```yaml
# Set vm.swappiness to 5 in /etc/sysctl.conf
- sysctl:
    name: vm.swappiness
    value: '5'
    state: present
```

**Comment by ansibot** [comment]
Files identified in the description:
* [`lib/ansible/modules/system/sysctl.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/sysctl.py)
* [`lib/ansible/modules/system/ufw.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/ufw.py)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @Akasurde @ahtik @davixx @felixfontein @ovcharenko @pyykkis
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @Akasurde @ahtik @davixx @felixfontein @ovcharenko @pyykkis
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by samdoran** [comment]
Updating the documentation is a good idea. However, I am not entirely happy how this warning is playing out. It is generating more confusion than help.

In hindsight, the way I should have implemented this was as an opt-in feature for string checking  of string types that module authors could leverage rather than warning on all string conversions, many of which are working just fine.

I will work on improving the behavior of this warning.

**Comment by felixfontein** [comment]
@samdoran maybe it would help if module authors could add a list of "types which are allowed to be converted"? So that if something expects an int or a port range (for example), conversion from int should be allowed (but not from float).

**Comment by samdoran** [comment]
@felixfontein I'm not sure. This all seems to be adding a lot of noise and complexity for very little gain.

**Comment by nirvdrum** [comment]
Given the issue turned into a bit of a meta-issue about whether the warning should exist at all, I don't think it should have been closed by an otherwise unrelated commit. #63505 fixed a documentation issue in a completely different module.

**Comment by samccann** [comment]
I'm working on finding the PR that addresses the meta issue to link to here.

**Comment by felixfontein** [comment]
@samccann it's #62027 by @samdoran

**Comment by samdoran** [comment]
@nirvdrum This issue was about `ufw` and `sysctl` specifically, which #63505 and #57984 fixed. 

As you stated, there are likely others that could be changed as well. Please open a new issue with a checklist of additional modules. Otherwise we end up with a very open ended issue without a clear scope of work.

The long term plan is this will become a non-issue once #62027 is merged and we go back to the way `str` types behaved before.

**Comment by nirvdrum** [comment]
That's fair. I had considered auditing all of the documentation for such changes to be part of the solution. I had thought that extended beyond just `ufw` and `sysctl`, but I admit I didn't do the legwork to verify that. But, I see there were several changes to solve the problem and they just hadn't gotten linked back to this one. I was pretty confused at first getting a notification that this one had been closed out due a changeset for `ufw`.

So, if the warnings are gone for everything, awesome. And I apologize for insinuating otherwise. As far as I had known, we were still where we were on Aug. 13.
